cilium-health/launch/endpoint.go

```diff
@@ -107,6 +108,42 @@ func configureHealthRouting(netns, dev string, addressing *models.NodeAddressing
 	return err
 }
 
+func configureHealthInterface(netNS ns.NetNS, ifName string, ip4Addr, ip6Addr *net.IPNet) error {
+	return netNS.Do(func(_ ns.NetNS) error {
```

Comment:
Did you check whether any function call inside the closure does not spawn a goroutine (ctx: https://www.weave.works/blog/linux-namespaces-golang-followup)?

Reply:
Yes, I did check for that and didn't find any suspicious code which spawns goroutines. I based my code on very similar code from `pkg/endpoint/connector`, which also invokes similar `netlink` commands in a `netNS.Do()` callback.